_layouts/home.html

```diff
@@ -18,4 +18,24 @@
       </li>
     {% endfor %}
   </ul>
+
+  <br>
+  <h3>Number of downloads for the gui version:</h3>
+  <div id="downloads"></div>
+  <script>
+    function createElement(version1, version2) {
```

Comment:
Could you rename version1 and version2 to something like win_version and linux_version ?